scripts/load_testing/create_account.py

```diff
@@ -38,7 +38,12 @@ def signup(self):
         resp = self.client.get(link, auth=auth)
         resp.raise_for_status()
         dom = pyquery.PyQuery(resp.content)
-        confirmation_token = dom.find('input[name="confirmation_token"]')[0].attrib['value']
+
+        if dom.find('input[name="confirmation_token"]')[0].attrib['value']:
+            confirmation_token = dom.find('input[name="confirmation_token"]')[0].attrib['value']
+        else:
+            print(dom)
+
```

Comment:
I like what you're trying to do here, but I think we may have some better ways. For general debugging,  you can run these from the command line with the addition of __main__ (see my branch). When you do that, you can insert PDB and get a good feel for what's happening and debug it.

If it's more about general error catching and reporting up, then what do you think of a try/except, reporting up the error (maybe in conjunction with dom)? Something like:

```
try:
    confirmation_token = dom.find('input[name="confirmation_token"]')[0].attrib['value']
except Exception(e):
     print(e)
```
It's a bare exception, but it'll get at what went sideways. Thoughts?

Reply:
The only hesitation with throwing an exception is that it may stop the locust run completing when the error may be a result of the high load. I'll test it out.

Reply:
I have an idea, how about a feature flag that will break to `pdb` upon error. Sometimes we'll want that on to debug and improve the load test, other times we'll want it off when we'll expect some errors due to the high load but we wont' want to interrupt the traffic/test.

Reply:
I like that idea. I resolved this to print something if the config is wrong in my PR, but I think a feature flag to go into pdb would be a nice add later.